news/lambdaclass-donates-100k-to-the-rust-foundation.md

```diff
@@ -0,0 +1,34 @@
+---
+title: LambdaClass Donates $100k to the Rust Foundation
+byline: Rust Foundation Team
+description: The Rust Foundation received a $100K USD donation from LambdaClass (“Lambda”).
+date: 2024-08-22T12:00:00Z
+tags:
+  - announcement
+  - foundation
+index: false
+layout: layouts/news.njk
+---
+The Rust Foundation is excited to announce a recent $100K USD donation from [LambdaClass](https://lambdaclass.com/) (“Lambda”) — a deep tech venture studio and investment firm with expertise in a variety of industries, including gaming, machine learning, cryptography, database design, distributed systems, programming languages, compiler development, high-frequency trading, real-time bidding, embedded devices, aerospace, and biotech.
+
+This donation was made by Lambda to “support the ongoing development and maintenance of the Rust programming language” – our core mission as the nonprofit steward of the Rust programming language and community.
+
+In a statement of intent about their donation, Lambda said the following:
+
+LambdaClass is dedicated to advancing the technological frontier by applying cutting-edge technologies to generate and disseminate scientific knowledge with practical applications. Rust has been an integral part of our work since 2014, delivering great results, and we believe our contribution will further strengthen the language's impact on the global tech community.
```

Comment:
should we this be a quote?

Reply:
I think this was intentional but I have pinged Gracie to double check. Thanks Marco.